src/com/palmergames/bukkit/towny/object/TownyObject.java

```diff
@@ -1,13 +1,14 @@
 package com.palmergames.bukkit.towny.object;
 
 import com.palmergames.bukkit.towny.TownyFormatter;
+import com.palmergames.bukkit.towny.object.metadata.CustomDataField;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
Remove wildcard import.

Reply:
Right. Different machine, so my settings weren't the same. I'll fix that now as well.